conda_forge_tick/mamba_solver.py

```diff
@@ -9,14 +9,19 @@
 Most of the code here is due to @wolfv in this gist,
 https://gist.github.com/wolfv/cd12bd4a448c77ff02368e97ffdf495a.
 """
-import json
+import rapidjson as json
```

Comment:
thought for another PR: should we just shim this globally?

Reply:
I don't have a strong opinion here.